components/Tabs/Tabs.js

```diff
@@ -1,88 +1,92 @@
+      this.parent.updateActive(this);
+      // using a jQuery method, add a class to this.element named "active-tab"
+      this.element.addClass("active-tab");
+      // iterate over each card using the .each() method in jQuery. call the selectCard() method in your callback function
+      this.cards = this.parent.each(this.element.selectCard('card'));
+    }
 
-  deselectTab(){
-    // use a jQuery method to remove the class "active-tab" from this.element
-    this.element;
-    // iterate over each card using the .each() method in jQuery. call the deselectCard() method in your callback function
-    this.cards;
+    deselectTab() {
+      // use a jQuery method to remove the class "active-tab" from this.element
+      this.element;
+      // iterate over each card using the .each() method in jQuery. call the deselectCard() method in your callback function
+      this.cards;
+    }
   }
-}
 
-class Tabs {
-  constructor(element){
-    this.element = $(element);
-    
-    // Using jQuery, find all of the tabs on this element.
-    this.tabs;
+  class Tabs {
+    constructor(element) {
+      this.element = $(element);
 
-    this.tabs = this.tabs.map( (i, tab) => new TabLink(tab, this) );
-    
-    // Set the initial active tab to the first tab in the list.
-    this.activeTab;
+      // Using jQuery, find all of the tabs on this element.
+      this.tabs;
 
-    this.init();
-  }
+      this.tabs = this.tabs.map((i, tab) => new TabLink(tab, this));
 
-  init(){
-    // use activeTab to call the selectTab() method
-    this.activeTab;
-  }
+      // Set the initial active tab to the first tab in the list.
+      this.activeTab;
 
-  updateActive(tabElement){
-    // use activeTab to call the deselectTab() method
-    this.activeTab;
-    // assign activeTab to tabElement
-    this.activeTab;
-  }
+      this.init();
+    }
 
-  getCards(data){
-    // This method is meant to get all the cards from the HTML page.  
-    // If the data supplied is 'all' then all of the cards should be returned. 
-    // Otherwise, only cards matching the data attribute should be returned. 
-  }
-}
+    init() {
+      // use activeTab to call the selectTab() method
+      this.activeTab;
+    }
 
-// Using jQuery, select the correct tabs component. Then initialize the Tabs class.
-let tabs = $();
-// tabs = new Tabs(tabs)
+    updateActive(tabElement) {
+      // use activeTab to call the deselectTab() method
+      this.activeTab;
+      // assign activeTab to tabElement
+      this.activeTab;
+    }
 
+    getCards(data) {
+      // This method is meant to get all the cards from the HTML page.  
+      // If the data supplied is 'all' then all of the cards should be returned. 
+      // Otherwise, only cards matching the data attribute should be returned. 
+    
+    }
+  }
 
+  // Using jQuery, select the correct tabs component. Then initialize the Tabs class.
+  let tabs = $(".");
```

Comment:
This line of code is causing an error on the whole page, need to specify the name of the class you are attempting to select with jQuery.